packages/strapi-plugin-users-permissions/admin/src/lifecycles/didGetSecuredData.js

```diff
@@ -0,0 +1,21 @@
+const pluginId = require('../pluginId');
+
+module.exports = function didGetSecuredData() {
+  const {
+    props: { updatePlugin },
+  } = this;
+  const leftMenuSections = [
+    {
+      links: [
+        {
+          label: 'Users',
+          destination: 'user',
+          plugin: 'content-manager',
```

Comment:
pluginId

Reply:
Well, the pluginId is equal to `users-permissions` in this file or we want to add some data in the `content-manager` main object which is responsible of the main menu.

This code will be outdated when we setup the menu api.